decko/lib/decko/rest_spec_helper.rb

```diff
@@ -0,0 +1,14 @@
+# -*- encoding : utf-8 -*-
+
+module Decko
+  # For use in REST API specs
+  module RestSpecHelper
+    def self.describe_api &block
+      RSpec.describe CardController, type: :controller do
+        routes { Decko::Engine.routes }
+        include Capybara::DSL
+        instance_eval(&block)
```

Comment:
Can't we do this with a shared context that includes `RSpec::Rails::ControllerExampleGroup`?

Reply:
I could believe it, but that will have to wait until after vacation :)